app.py

```diff
@@ -129,12 +128,44 @@ def _register_indexes(graph: GraphDatabase, index_base_url: str, dry_run: bool =
     return index_urls
 
 
-def _do_schedule_core_solver_jobs(
-    openshift: OpenShift,
-    index_urls: List[str],
-    package_name: str,
-    package_version: str,
-    output: str,
+def _take_data_science_packages() -> List[str]:
+    """Take list of Python Packages for data science."""
+    current_path = Path.cwd()
+    complete_file_path = current_path.joinpath("hundredsDatasciencePackages.yaml")
+
+    with open(complete_file_path) as yaml_file:
+        requested_packages = yaml.safe_load(yaml_file)
+
+    data_science_packages = []
+    for package_name in requested_packages["hundreds_datascience_packages"]:
+        _LOGGER.info("Registering package %r", package_name)
+        data_science_packages.append(package_name)
+
+    return data_science_packages
+
+
+def _schedule_default_packages_solver_jobs(packages: List[str], index_urls: List[str]) -> None:
+    """Run solver jobs for Python packages list selected."""
+    openshift = OpenShift(kubernetes_verify_tls=False)
```

Comment:
`kubernetes_verify_tls=False` should not be hardcoded - if it is needed, let's have it inside the template as an environmnent variable - ``KUBERNETES_VERIFY_TLS=0``.

Reply:
Forgot to change it, I was testing one thing, thanks!!